packages/cli/bin/setup.js

```diff
@@ -41,9 +41,12 @@ async function setup() {
   if (currentTermId.eq(bn(0)) && neededTransitions.eq(bn(0))) {
     logger.warn('Protocol has not started yet, please make sure Protocol is at term 1 to create disputes and run the script again.')
   } else {
+
     // create disputes
     for (let i = 0; i < disputes; i++) {
-      const arbitrable = arbitrables[i]
+      const output = execSync(`node ./bin/index.js arbitrable -n ${network}`)
```

Comment:
Wasn't sure, but does this deploy a new `ArbitrableMock`? Why aren't we able to use the previous `arbitrables[i]`?

Looking at the old code, was this just broken code?